src/main/java/net/mcreator/minecraft/link/CurrentDevice.java

```diff
@@ -151,8 +151,18 @@ public static void disableDigitalInputEvents(int port) {
 	 */
 	public static void sendMessage(String message) {
```

Comment:
Here you should add another method to send message and data, and not parse the data here

Reply:
Done, also made a little change in command handling for data to be optional

Reply:
Method names can be same as you can overload them

Reply:
I cannot overload them, because they have identical parameters
![image](https://user-images.githubusercontent.com/83773843/133656807-153e5425-e552-4be3-9dee-1f6ee8b2d8dd.png)

Reply:
The second one should accept two arguments instead of parsing them from one

Reply:
Done, overloaded this function and then parsed arguments in command section